INSTALL.rst

```diff
@@ -40,7 +47,7 @@ Disable SELinux
 ---------------
 
 The setup described below runs into problems with SELinux related to the sqlite
-database. For now the recommended solution is to simply disable SELinux. In
+database and apache. For now the recommended solution is to simply disable SELinux. In
```

Comment:
I don't think we fully understand exactly what the problem SELinux is having is, and I'd rather not mis-document it. With sqlite it was pretty clear what the specific problem was, it's less immediately obvious what's causing it on a postgres setup.